src/xhr.js

```diff
@@ -387,6 +387,15 @@ class FetchResponse {
   }
 
   /**
+   * Drains the response and returns a promise that resolves with the response
+   * text.
+   * @returns {!Promise.<string>}
```

Comment:
qq, is it `@returns` or `@return`? https://developers.google.com/closure/compiler/docs/js-for-compiler#tags says it should be `@return`